src/atoms/Button/buttons.module.scss

```diff
@@ -107,6 +107,12 @@ $button-solid-background-tertiary-color:   #f08f5c;
   color: $button-info-color;
 }
 
+.slim {
+  padding-top: ru(.5);
+  padding-bottom: ru(.5);
+  line-height: 1em;
```

Comment:
Is there a reason this isn't rem?

Reply:
not sure, I just copied it from one of the other variants - I'll see if I can convert them both to rem